### ISSUE TITLE
Add STT label

### ISSUE THREAD

**Issue by benetechMatt** [benetech/MathShare#815]
Add a label underneath that says "Speech to text", "voice recognition", or "record voice" could be helpful for students unfamiliar with it.  

_Originally posted by @benetechMatt in https://github.com/benetech/MathShare/issues/745#issuecomment-545675088_

Aha! Link: https://diagramlabs.aha.io/features/B-106

**Comment by benetechMatt** [comment]
one word labels could be

- dictate
- speak
- record
- transcribe

**Comment by johnhbenetech** [comment]
We will also need copy for screen reader alerts, per James:

"In terms of screen reader support, it would probably be a good idea to indicate, via a live region-style status update, when audio capture begins and ends. The accessible label of the button would also need to be updated to distinguish between the capturing and idle states."

And separate labels for pre record and active record

**Comment by benetechMatt** [comment]
pre-record: speak
Active-record: listening

**Comment by jscholes** [comment]
@benetechMatt - a button with an accessible label of "Speak" doesn't necessarily imply dictation.  I'd be more inclined to think that the website would speak when I pressed it, not that I should.

For consistency and to create a predictable UI, the label ideally shouldn't change from an actionable verb to a continuous state.

**Comment by benetechMatt** [comment]
After some discussion we decided this makes the most sense and avoids confusion. for non-screen reader students that don't understand the word "dictate" they will likely recognize the microphone icon as it's widely used on smart phones.

Pre-record: dictate
active recording: talk now...

**Comment by johnhbenetech** [comment]
@benetechMatt what about for the aria live region for screen readers. We will need copy for essentially 
dictation has begun' and 'dictation complete'.

**Comment by jscholes** [comment]
@benetechMatt - it doesn't make sense for a dictation user to say "Talk Now" once they've finished, if e.g. they're activating the button to stop dictation using a third party speech input tool like Dragon.

**Comment by benetechMatt** [comment]
I think I understand the process then.  The button should then say "stop dictation" so the user can select/click that button

**Comment by jscholes** [comment]
@benetechMatt precisely.

**Comment by benetechMatt** [comment]
@jscholes for speech input tools like Dragon, why would they use our dictation tool instead of using their software's dictation directly to the text area?

**Comment by jscholes** [comment]
@benetechMatt - to a certain extent it depends how well their system-wide dictation tool understands mathematic input, but also the reasons why Benetech had this feature implemented in the first place.  If it was only intended for those without access to Dragon or whatever already, and has no additional MathShare-specific features, then they probably wouldn't.

But then that raises questions of how a user who relies on speech input would activate your record button, allow microphone permissions within the browser, or even open the app in the first place.  If they have no other speech input software, they might not even know how to talk to such a tool.  Once the STT button has been activated, does MathShare provide a command to stop audio capture and is this clearly signposted?

**Comment by johnhbenetech** [comment]
@rupeshparab Let's start with adding text label beneath the microphone icon
"Dictate" before recording
"Stop Dictation" during recording

We should also fire off an aria live alert - 'Dictation started' and then after 'Dictation complete' once stopped.
 Once we have this live we can circle back with @benetechMatt and @jscholes for feedback

**Comment by rupeshparab** [comment]
@johnhbenetech 
I have added dictation labels and aria-live announcement

Also, commit cb6b9e4 is related to #793, I tagged it incorrectly to this story

**Comment by jscholes** [comment]
@rupeshparab - the `<img>` within the button has alt text of "start speaking", meaning that the accessible label for the button is either "Start Speaking Dictate" or "Start Speaking Stop Dictation".  The image should be marked as decorative, via an empty `alt` attribute.

**Comment by rupeshparab** [comment]
I have set empty string in alt attribute

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/b0b8e8330b582d639c293a6d0249c9a7ab96b165.
